apps/cowswap-frontend/src/common/hooks/useUnsupportedNetworksText.tsx

```diff
@@ -1,21 +1,12 @@
 import { ReactElement } from 'react'
 
 import { getChainInfo } from '@cowprotocol/common-const'
```

Comment:
You need to remove this import as well, breaking the linter:

<img width="934" height="96" alt="Image" src="https://github.com/user-attachments/assets/02210563-140e-489d-8544-9765fcf87d54" />